src/context/fetch.ts

```diff
@@ -26,6 +26,21 @@ export const augmentRequestInit = (requestInit: RequestInit): RequestInit => {
   }
 }
 
+const createFetchRequestParameters = (input: MockedRequest) => {
+  const { body } = input
+  const requestParameters: RequestInit = {
+    ...input,
+    body: null,
```

Comment:
I believe that the default value of the request `body` is `undefined` when it's not set.

This is also how we are handling a GET request body received from the Service Worker:

https://github.com/mswjs/msw/blob/c254c99c9a658e5619ace45cbf03ddeb1fbd027b/src/utils/handleRequestWith.ts#L39-L50